main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-steputils/v2/stepenv"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-steplib/tmp-bitrise-step-create-new-app-store-release/step"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
```

Comment:
We have the exitcode package for exit statuses, example usage: https://github.com/bitrise-steplib/steps-fastlane/blob/master/main.go#L22